Library/Formula/ircd-hybrid.rb

```diff
@@ -1,8 +1,8 @@
 class IrcdHybrid < Formula
   desc "High-performance secure IRC server"
   homepage "http://www.ircd-hybrid.org/"
-  url "https://downloads.sourceforge.net/project/ircd-hybrid/ircd-hybrid/ircd-hybrid-8.2.11/ircd-hybrid-8.2.11.tgz"
-  sha256 "eeb916ca22b0ee2c7b36e00249ca41001a82587597084c102fe11ad8e85cfe78"
+  url "https://github.com/ircd-hybrid/ircd-hybrid/archive/8.2.12.zip"
```

Comment:
We could use `.tar.gz` instead of `.zip` to get a smaller file (~1Mo vs. ~1.3Mo).